llm_studio/src/datasets/text_rlhf_modeling_ds.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Any, Dict
+
+import numpy as np
+import pandas as pd
+import torch
+
+from llm_studio.src.datasets.text_causal_language_modeling_ds import (
+    CustomDataset as CausalLMCustomDataset,
+)
+from llm_studio.src.datasets.text_utils import get_texts
+
+logger = logging.getLogger(__name__)
+
+
+class CustomDataset(CausalLMCustomDataset):
+    def __init__(self, df: pd.DataFrame, cfg: Any, mode: str = "train"):
+        assert (
+            cfg.dataset.system_column == "None"
+        ), "RLHF is not compatible with system column."
```

Comment:
This is for CLI usage? Shall we just ignore it, or would you still throw the error?

We should also open a new issue to allow system cols with RLHF. The issue is the reward model, so probably depends on the custom reward model training first.

Reply:
Yeah, it is basically an additional safety check.